### DISCUSSION TITLE
ADempiere Steering meeting July 22nd, 2022

### DISCUSSION THREAD

**Discussion by rvergara** [adempiere/adempiere#3953] [Steering Meetings]
### Meeting Info
**Date:** July  22, 2022
**Time:** 19:00 GMT ~ 20:00 GMT
**Location**: Discord meeting room
**Meeting URL:** None
**Recorder:** No
**Meeting Objetive:** Review of Project and admin activities to be performed
### **Participants**
@rvergara 
@niclopup 
@e-Evolution 
@eduardolopezuy 
@yamelsenih 
@marcalwestf 
### **Agenda**

1. Progress ADempiere web site
2. Knowledge base tools discussions
3. 3.9.4 release calendar discussion
4. Others
### **Points Discussied**

- ADempiere web site in Enlish is almost complete, collaborators are reviewing final texts and adding professional images, ADempiere web site in spanish is in progress and it is expected to be completed this week (July 29th)
- In order to expedite the progress of the web site both in English and spanish, R. Vergara requested written privileges for the adempiere-site repository and this was approved by the ADempiere management group (Mario, Victor and Yamel)
- Yamel indicated that for next meeting he will be presenting a knowledge tool proposal, Discussion on this matter was reschedule for next steering meeting (August 5th)
- 3.9.4 last changes required for final version release will be completed by Yamel on this week (July 29 th)
- a quick review of Nicolas Lopez proposal for discord channel management was performed. Nicolas has attached the proposal document to this discussion for steering members perusal. A decisión on how to proceed on this matter will be taken on next steering meeting (August 5th)

### **Next Meeting**
August 5th. 19:00 GMT, Discord meeting room

**Comment by niclopup** [comment]
I would like to discuss the rules and criteria to define in the Community ADempiere Discord Server in order to use only one to all the community with a clear a unique way to communicate to the community

After analyzing some others servers of communities in Discord i suggest the following proposal:
https://docs.google.com/document/d/1-0iWdyE_UQzCYTueYc1O8kjczzwwh6nz6pj5pz_KzwI/edit
